datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```diff
@@ -17,7 +17,7 @@ clickhouse-driver==0.2.0; python_version < '3.0'
 clickhouse-driver==0.2.3; python_version > '3.0'
 contextlib2==0.6.0.post1; python_version < '3.0'
 cryptography==3.3.2; python_version < '3.0'
-cryptography==3.4.8; python_version > '3.0'
+cryptography==38.0.3; python_version > '3.0'
```

Comment:
![image](https://user-images.githubusercontent.com/611228/207586533-32cdc4e1-8e81-4c73-8f3c-df5f89eaa1b7.png)
I wonder what happened there

Reply:
They changed the versioning scheme starting with that release (https://cryptography.io/en/latest/changelog/#v35-0-0, https://cryptography.io/en/latest/api-stability/#versioning).